src/cljs_tooling/util/analysis.cljc

```diff
@@ -137,10 +137,29 @@
         excludes (:excludes (find-ns env ns))]
     (apply dissoc macros excludes)))
 
+(def ^:private common-keywords
```

Comment:
Not sure that `common` is the right term here. Not sure how exactly did you come up with the list - might be a good idea to explain this in the docstring.

Reply:
From `lumo`, and it is arbitrary. I guess in Clojure there is a similar list somewhere?

Reply:
Yeah, I just think it'd be better to use this list directly, instead of hardcoding something ourselves, but that's not a big deal.